R/frontline_dm.R

```diff
@@ -0,0 +1,149 @@
+#' @title Parse frontline date
+#' @param format string. The format to use to parse the string. Default is
+#' '%m.%d.%y'.
+#' @return date
+parse_date.frontline <- function(x, format = "\\d{4}-\\d{2}-\\d{2}"){
+
+  date <- lubridate::ymd(stringr::str_extract(x, format))
+}
+
+#' @title Data management for date column
+#' @description Manage date
```

Comment:
@sadierichardson let's add some detail about why we filter the date to be less than the export_date in the description

Reply:
@holmesjoli done! let me know if that makes sense.